src/main.rs

```diff
@@ -1,5 +1,5 @@
 // disable console opening on windows
```

Comment:
Very small nit-pick: would you mind extending the comment avoe to something like "disable console on windows for release builds"?